systems/plants/RigidBody.h

```diff
@@ -108,7 +108,7 @@ class DLLEXPORT_RBM RigidBody {
   MatrixXd ddTdqdq;
 
   double mass;
-  Vector4d com;  // this actually stores [com;1] (because that's what's needed in the kinematics functions)
+  Vector3d com; 
```

Comment:
After rebasing my changes onto your branch, I ran into some trouble related to this variable when running compareParsers.There's definitely a mistake in RigidBodyManipulatorURDF.cpp, line 222:
```
  body->com << T(0, 3), T(1, 3), T(2, 3), 1.0;
```
Maybe it's because I'm in debug mode, but it's weird that compareParsers doesn't cause a segfault on the build servers.